src/wallet/wallet_rpc_server_commands_defs.h

```diff
@@ -1849,15 +1849,15 @@ namespace wallet_rpc
     struct request
     {
       BEGIN_KV_SERIALIZE_MAP()
-      END_KV_SERIALIZE_MAP()
+        END_KV_SERIALIZE_MAP()
```

Comment:
What is the deal with all these whitespace changes? Is this from upstream? Surely the original makes more sense.

Reply:
Yep, I've cleaned it up. My bad